README.md

```diff
@@ -3,26 +3,6 @@ Mailphax
 
 Send faxes with Phaxio using 3rd party email services.  Mailphax is a simple sinatra app.  You can run it on any host or with any service that supports ruby and sinatra.
 
-
-Installation on Heroku
```

Comment:
Botch pull-request branch. This was intended for strictly a work-related project. Undone in #3.